docs/app/pages/components/components-sections/media-player/media-player.module.ts

```diff
@@ -33,11 +33,10 @@ const ROUTES = [
     ],
     exports: SECTIONS,
     declarations: SECTIONS,
-    entryComponents: SECTIONS
 })
 export class ComponentsMediaPlayerModule {
```

Comment:
Getting an error in this section:

![image](https://user-images.githubusercontent.com/20795331/75800429-72fb9780-5d71-11ea-98e3-21799e637ab9.png)

I'm guessing AudioService can be made providedIn: root